README.md

```diff
@@ -28,6 +28,12 @@ cd itchy-cordova
 npm install
 ```
 
+For styles we are using LESS, add it globally:

```

Comment:
This is not needed, as we use a bundling tool, parcel, which compiles LESS automatically with it just being linked in the HTML like a CSS file. That also means you should remove the compiled CSS from the asset folder

Reply:
Oh okey, so can I also remove the lessc compiler from the build-styles tag??

What about autoprefixer plugin is being added by parcel too?

Reply:
I believe Parcel should handle autoprefixing as well, I'll look into it when I get a chance.  Thanks so much for your contribution!